### PR TITLE
Ddp 7967 social media link updates

### PR DESCRIPTION
**Pull request by grigol-brd**

### CODE REVIEW

File: ddp-workspace/projects/ddp-brain/src/app/app.module.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ import { MatButtonModule } from '@angular/material/button';
 import { MatExpansionModule } from '@angular/material/expansion';
 import { ModalImageComponent } from './components/modal-image/modal-image.component';
 import { PrismComponent } from './components/prism/prism.component';
+import {tkCfg} from "../../../ddp-angio/src/app/app.module";
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-brain/src/app/app.module.ts]: unused import

File: ddp-workspace/projects/ddp-mpc/src/app/app.module.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ import { WelcomeComponent } from './components/welcome/welcome.component';
 import { MoreDetailsComponent } from './components/more-details/more-details.component';
 import { AboutUsComponent } from './components/about-us/about-us.component';
 import { DataReleaseComponent } from './components/data-release/data-release.component';
+import {tkCfg} from "../../../ddp-angio/src/app/app.module";
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-mpc/src/app/app.module.ts]: unused import

**Comment by alextanlan** [comment]
Can you please fix linter warnings ?

**Comment by grigol-brd** [comment]
> Can you please fix linter warnings ?

**Comment by grigol-brd** [comment]
> Can you please fix linter warnings ?

already did

**Comment by alextanlan** [comment]
> already did

not exactly. please see 
https://github.com/broadinstitute/ddp-angular/pull/1307/checks?check_run_id=6591666788
https://app.circleci.com/pipelines/github/broadinstitute/ddp-angular/16092/workflows/687435b9-70fd-4d4c-b64b-3b4ae9228e54/jobs/24540

**Review by alextanlan — APPROVED** [review]

**Comment by grigol-brd** [comment]
> > already did
> 
> not exactly. please see https://github.com/broadinstitute/ddp-angular/pull/1307/checks?check_run_id=6591666788 https://app.circleci.com/pipelines/github/broadinstitute/ddp-angular/16092/workflows/687435b9-70fd-4dno4c-b64b-3b4ae9228e54/jobs/24540
DONE

**Pull request merged**